kubetest2-tf/deployer/deployer.go

```diff
@@ -61,6 +63,10 @@ type deployer struct {
 	provider      providers.Provider
 }
 
+func (d *deployer) Version() string {
+	return GitTag
```

Comment:
where are we setting this variable?

Reply:
@mkumatag We set this variable while building the binary..
Like in https://github.com/kubernetes-sigs/kubetest2/blob/master/Makefile#L65